src/gui/filedetails/shareemodel.cpp

```diff
@@ -275,4 +342,13 @@ void ShareeModel::filterSharees()
     Q_EMIT shareesReady();
 }
 
+void ShareeModel::slotDarkModeChanged()
+{
+    for (int i = 0; i < _sharees.size(); ++i) {
+        if (_sharees[i]->updateIconUrl()) {
+            Q_EMIT dataChanged(index(i), index(i), {IconRole});
+        }
+    }
```

Comment:
Why not just call `dataChanged` once for all the rows in the model?

Reply:
@claucambra I was not able to come up with a proper call as per your suggestion. Do you have an example? Whatever I try fails to update the view as a result.